ktor-server/ktor-server-tests/jvm/test/io/ktor/tests/server/features/CORSTest.kt

```diff
@@ -670,4 +670,29 @@ class CORSTest {
             }
         }
     }
+
+    @Test
+    fun testAnyHeader() {
+        withTestApplication {
+            application.install(CORS) {
+                anyHost()
+                anyHeader()
+            }
+
+            application.routing {
+                get("/") {
+                    call.respond("OK")
+                }
+            }
+
+            handleRequest(HttpMethod.Options, "/") {
```

Comment:
Two cases need to be tested including missing `AccessControlRequestHeaders` in the request.